src/common/Loader/index.js

```diff
@@ -1,7 +1,7 @@
 import { Header, LoadingIcon } from "./styled";
 import { useQueryParameter } from "../useQueryParameter";
 import { searchQueryParamName } from "../queryParamNames";
-import { Wrapper } from "../Wrappers/LoaderNoResultsWrapper";
+import { Wrapper } from "../StyledComponents/LoaderNoResultsWrapper";
```

Comment:
its ok with not using index.js (it would be the default file in than catalog - we dont need to use him now.
Wrappers renamed to styledCommonComponents